# returns the bodyid given its name
string name
---
int32 bodyid
